ta/pkcs11/src/pkcs11_token.c

```diff
@@ -35,11 +35,14 @@
  *
  * @link - chained list of registered client applications
  * @sessions - list of the PKCS11 sessions opened by the client application
+ * @token_object_handle_db - Database for token object handles published by the
+ * shared sessions
  */
 struct pkcs11_client {
 	TAILQ_ENTRY(pkcs11_client) link;
 	struct session_list session_list;
 	struct handle_db session_handle_db;
+	struct handle_db token_object_handle_db[TOKEN_COUNT];
```

Comment:
Why are we having the object_handle_db per token ? If this is required, shouldn't the same logic also apply to session_handle_db in this structure ?